jest.config.ts

```
@@ -6,7 +6,7 @@ const config: Config = {
   transform: {
     '^.+\\.(ts|tsx)$': ['ts-jest', {
       tsconfig: {
-        jsx: 'react',
+        jsx: 'react-jsx',
```

ESLint

[error] Insert `··`

(prettier/prettier)